rkt/metadatasvc.go

```diff
@@ -129,7 +168,7 @@ func handleRegisterApp(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	m, ok := metadataByUID[*uid]
+	c, ok := containerByUID[*uid]
```

Comment:
It would be great if this kind of refactoring took place in a separate pull-request.